packages/astroport_periphery/Cargo.toml

```
@@ -24,15 +24,13 @@ overflow-checks = true
 backtraces = ["cosmwasm-std/backtraces"]
 
 [dependencies]
-cw20 = { version = "0.8" }
-
-terra-cosmwasm = { version = "2.2.0" }
-cosmwasm-std = { version = "0.16.0", features = ["iterator"] }
-cosmwasm-bignumber = "2.2.0"
-terraswap = "2.4.0"
-
+cw20 = { version = "0.13.2" }
+# terra-cosmwasm = { version = "2.2.0" }
+cosmwasm-std = { version = "1.0.0" }
```

```suggestion
cosmwasm-std = "1.0"
```

Use 1.0 so cargo will be able to update patch versions